src/frequenz/sdk/timeseries/_voltage_streaming.py

```diff
@@ -0,0 +1,188 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Fetch and stream the phase-to-neutral voltage from a source component.
+
+Each phase of the phase-to-neutral voltage is fetched from the source
+component individually and then streamed as a 3-phase sample.
+"""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import TYPE_CHECKING
+
+from frequenz.channels import Receiver, Sender
+
+from ..actor import ChannelRegistry
+from ..microgrid import connection_manager
+from ..microgrid.component import Component, ComponentCategory, ComponentMetricId
+from ..timeseries._base_types import Sample, Sample3Phase
+from ..timeseries._quantities import Quantity, Voltage
+
+if TYPE_CHECKING:
+    # Imported here to avoid a circular import.
+    from ..actor import ComponentMetricRequest
+
+_logger = logging.getLogger(__name__)
+
+
+class VoltageStreaming:
```

Comment:
```suggestion
class VoltageStreamer:
```
?

Reply:
Makes sense. Added a new commit to rename the class and related files.